src/App.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import './App.css';
 import BasicExample from './BasicExample';
+import configureStore from './stores/Store';
```

Comment:
Đổi tên file Store.js -> index.js
Khi này chỉ cần

```suggestion
import configureStore from './stores/';
```